meta/main.yml

```diff
@@ -6,7 +6,7 @@ galaxy_info:
     Role to install the python package openstacksdk
   company: StackHPC Ltd
   license: Apache2
-  min_ansible_version: 2.5
+  min_ansible_version: 2.9
```

Comment:
Why 2.9?

Reply:
Oldest non-EOL release

Reply:
It's not really up to this role to dictate that though. Distros may provide support for older releases